### ISSUE TITLE
Validate Porter with Artifactory

### ISSUE THREAD

**Issue by carolynvs** [getporter/porter#2324]
We've received verification that Artifactory works with CNAB in https://github.com/cnabio/cnab.io/issues/58. Let's check quick using porter, by running the following commands:

```
porter explain --reference ghcr.io/getporter/examples/porter-hello:v0.2.0
porter copy --source ghcr.io/getporter/examples/porter-hello:v0.2.0 --destination ARTIFACTORY_BUNDLE_REFERENCE
```

and then make sure we can publish

```
mkdir test && cd test
porter create
porter publish --registry ARTIFACTORY_REGISTRY
porter install --reference ARTIFACTORY_BUNDLE_REFERENCE
```

Once we've verified that it works with Porter, update our [compatible registries page](https://github.com/getporter/porter/blob/release/v1/docs/content/compatible-registries.md) to list that it works.

You can preview the documentation change using the built-in netlify preview generated when you submit your pull request.

See our [Contributing Tutorial](https://getporter.org/contribute/tutorial) and [New Contributor Guide](https://github.com/getporter/porter/blob/release/v1/CONTRIBUTING.md) for help getting started contributing to Porter.

**Comment by MadhuMPandurangi** [comment]
Hi @carolynvs, I want to contribute to this issue. Can I be assigned ? It'll help me on gaining experience working on open source and it'll solve the issue.

**Comment by carolynvs** [comment]
Yes, please try out Porter 1.0.0-rc.1 with artifactory and let us know how it works out! 👍

**Comment by h20220025** [comment]
Hi @carolynvs, This issue seems to be a good initiation of my journey towards contribution to FOSS projects.
Can you please assign this to me?

@MadhuMPandurangi please confirm if you are not working on this.

**Comment by MadhuMPandurangi** [comment]
@h20220025 I'm working on issue #2228. No objection from my side 👍.

**Comment by carolynvs** [comment]
@h20220025 I would love to know if Artifactory works with porter v1. Please let me know what you find! 👍

**Comment by h20220025** [comment]
Thank You @carolynvs, could you please assign this issue to me.

**Comment by carolynvs** [comment]
GitHub can't assign issues to people who are not in the organization, but yes consider it assigned! ✅

**Comment by h20220025** [comment]
![artifactory_2](https://user-images.githubusercontent.com/112873674/202269828-7da55ae8-88c5-4293-acd3-cb0da52c9bed.png)

the second command gave some errors, how can these errors be fixed ??

**Comment by h20220025** [comment]
@carolynvs please guide me regarding this issue.

**Comment by carolynvs** [comment]
You will need to alter the commands that use the placeholder text `ARTIFACTORY_REGISTRY` and `ARTIFACTORY_BUNDLE_REFERENCE` to substitute the location of the bundle on the artifactory registry that you are testing against.

For example, if you test artifactory instance is located at localhost:5000, the commands should be 

```
porter explain --reference ghcr.io/getporter/examples/porter-hello:v0.2.0

porter copy --source ghcr.io/getporter/examples/porter-hello:v0.2.0 --destination localhost:5000/porter-hello:v0.20
```

and then

```
mkdir test && cd test
porter create
porter publish --reference localhost:5000/porter-hello:v0.1.0
cd /tmp # leave the bundle directory
porter install --reference localhost:5000/porter-hello:v0.1.0 
```

**Comment by h20220025** [comment]
thank you ,@carolynvs

**Comment by h20220025** [comment]
Hi @carolynvs, I tried
`porter copy --source ghcr.io/getporter/examples/porter-hello:v0.2.0 --destination localhost:5000/porter-hello:v0.20`

I'm getting error, connection refused. 

> error retrieving bundle metadata for localhost:5000/porter-hello:v0.20: Get "https://localhost:5000/v2/": dial tcp 127.0.0.1:5000: connect: connection refused; Get "http://localhost:5000/v2/": dial tcp 127.0.0.1:5000: connect: connection refused
> Copy stopped because detection of localhost:5000/porter-hello:v0.20 in the destination registry failed. To overwrite it, repeat the command with --force specified: error retrieving bundle metadata for localhost:5000/porter-hello:v0.20: Get "https://localhost:5000/v2/": dial tcp 127.0.0.1:5000: connect: connection refused; Get "http://localhost:5000/v2/": dial tcp 127.0.0.1:5000: connect: connection refused
> Copy stopped because detection of localhost:5000/porter-hello:v0.20 in the destination registry failed. To overwrite it, repeat the command with --force specified: error retrieving bundle metadata for localhost:5000/porter-hello:v0.20: Get "https://localhost:5000/v2/": dial tcp 127.0.0.1:5000: connect: connection refused; Get "http://localhost:5000/v2/": dial tcp 127.0.0.1:5000: connect: connection refused

**Comment by carolynvs** [comment]
What is the URL of the artifactory instance that you are testing against? I gave localhost:5000 as an example but yours is probably something different.

**Comment by h20220025** [comment]
Since I'm contributing for the first time can u help me to know the url of artifactory instance. And where to look at.

**Comment by carolynvs** [comment]
Sorry @h20220025, but the reason I opened this issue is because we needed help from someone who has or is willing to figure out how to set up Artifactory. 😁  I am unable to provide instructions on how to do that, so this isn't the best issue for you to do right now.

**Comment by h20220025** [comment]
@carolynvs thanks for support. Well, I am willing to do that.🙂

**Comment by h20220025** [comment]
@carolynvs could you please suggest some resources about this issue like how to work on artifactory bundles, so that I can work on that, you can suggest me some learning environments.
